restapi/src/test/java/io/stargate/web/docsapi/service/DocumentServiceTest.java

```diff
@@ -134,6 +137,11 @@ protected Schema schema() {
     return schema;
   }
 
+  @BeforeAll
+  public static void setupJackson() {
+    mapper.registerModule(new GuavaModule());
```

Comment:
Sure this is test related only? This does not have to be registered in the "prod" env usage?

Reply:
Integration tests passed :) I assume this is done automagically when the HTTP service starts. This here is just for unit tests that do not use any of our HTTP infrastructure.